src/drafts/MarkdownEditor/MarkdownEditor.tsx

```diff
@@ -403,7 +403,7 @@ const MarkdownEditor = forwardRef<MarkdownEditorHandle, MarkdownEditorProps>(
                 '&: focus-within':
                   view === 'edit'
                     ? {
-                        outline: '2px solid var(--borderColor-accent-emphasis)',
+                        outline: theme => `2px solid ${theme.colors.accent.emphasis}`,
```

Comment:
Optional suggestion, tiny bit cleaner:

```suggestion
                        outline: '2px solid',
                        outlineColor: 'accent.emphasis'
```